libs/crypto/src/factory.cpp

```diff
@@ -12,6 +12,10 @@
 // file.
 
 #include "crypto/factory.hpp"
+#ifdef USE_OPENSSL
```

Comment:
is this ifdef used outside of the crypto library?
if so, it should have a different name that reflects its usage for signatures.

Reply:
outside the crypto library the `factory.hpp` header `Factory::getSigner()` is used.
this ifdef is for crypto library compilation only